### PR TITLE
@damassi => Removing old artist app and renaming artist2 -> artist

### PR DESCRIPTION
**Pull request by mzikherman**
Took a little break from testing (hit the same snag with CV tab tests as the Filter, will pair w/ someone tomorrow), and did this little cleanup.

### CODE REVIEW

File: src/desktop/apps/artist/components/cta.js [hunk 1]
```diff
@@ -31,7 +32,13 @@ const send = {
 }
 
 if (sd.ARTIST_PAGE_CTA_ENABLED && sd.ARTIST_PAGE_CTA_ARTIST_ID) {
-  metaphysics(send).then(({ artist }) => {
-    cta(new Artist(artist))
+  metaphysics(send).then(({ artist: artistData }) => {
+    const artist = new Artist(artistData)
```
Comment (mzikherman) [inline src/desktop/apps/artist/components/cta.js]: This is a legit diff, I added this snippet to account for: https://github.com/artsy/force/blob/f1f35ef971ca3572696aa56306a09b923990e99b/src/desktop/apps/artist/client/cta.coffee (we're reusing the old CTA component overlaid on the new artist page), which we were using directly.

File: src/desktop/apps/artist/server.js [hunk 2]
```diff
@@ -47,10 +47,6 @@ app.get("/artist/:artistID*", async (req, res, next) => {
       !user && isExternalReferer && !IS_MOBILE
     res.locals.sd.ARTIST_PAGE_CTA_ARTIST_ID = req.params.artistID
 
-    // While we are rolling out the new page, override the default (`artist`)
```
Comment (mzikherman) [inline src/desktop/apps/artist/server.js]: This is the other legit diff/key update (nice job git on generating a nice diff, easy to see straightforward deletions and renames).

We remove the page type override from `new-artist` back to `artist` (cc @joeyAghion any updates to tracking or dashboards needed?). Also use the renamed asset package.

File: src/desktop/assets/artists_artworks.coffee [hunk 3]
```diff
@@ -1,9 +1,6 @@
 require('backbone').$ = $
 
 routes =
-  '/artist/.*': ->
-    require('../apps/artist/client/index.coffee').init()
-
```
Comment (damassi) [inline src/desktop/assets/artists_artworks.coffee]: Now taken care of via https://github.com/artsy/force/pull/3024/files#diff-24d93602cc2b0212be6294f5249cf65cR2, yeah?

File: src/desktop/assets/artists_artworks.styl [hunk 4]
```diff
@@ -26,7 +26,6 @@
 @require '../components/embedded_inquiry'
 @require '../components/inquiry_questionnaire/stylesheets'
 @require '../apps/inquiry/stylesheets'
-@require '../apps/artist/stylesheets'
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/desktop/assets/artists_artworks.styl]: Cool, now that we use styled-components shouldn't need it

**Comment by mzikherman** [comment]
> Would you mind removing the Artist backbone collection / model code as well? (Including tests, etc -- that is if there is any code since we used MP on the old pages)

@damassi I have to check, but I'm pretty sure we can't just remove the `Artist` backbone model. That's such a core model, that not only would it have been referenced in the removed artist app, but countless other places in Force as well. I'm not sure how much of that stuff is removable.

**Comment by damassi** [comment]
No prob 👍

File: src/desktop/assets/artists_artworks.coffee (hunk 3 above)
Comment (mzikherman) [reply src/desktop/assets/artists_artworks.coffee]: yup exactly

File: src/mobile/apps/partner_profile/templates/artist.jade [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
+-var name = artist.name || (typeof artist.get === 'function' ? artist.get('name') : null)
+
 block content
   .main-side-margin
     nav.slash-breadcrumbs
       a( href="/#{profile.get('id')}" )= profile.get('owner').name
       span
         a( href="/#{profile.get('id')}/artists") Artists
-  include ../../artist/templates/header
+  header#artist-page-header
```
Comment (mzikherman) [inline src/mobile/apps/partner_profile/templates/artist.jade]: This was also sketchy, this was sharing/rendering a file from the artist app. I just copied that into here.

File: src/desktop/apps/artist/server.js (hunk 2 above)
Comment (joeyAghion) [reply src/desktop/apps/artist/server.js]: I don't think anything is referencing `new-artist` directly, so this is 👍.

**Comment by mzikherman** [comment]
Ok, this is 💚 so going for the merge. We can keep it on staging for a little while to make sure.

**Pull request merged**